src/avatar_action.cpp

```diff
@@ -1000,20 +1001,25 @@ void avatar_action::use_item( avatar &you, item_location &loc )
             return;
         }
 
-        const item &it = *loc.get_item();
-        if( it.has_flag( "ALLOWS_REMOTE_USE" ) ) {
-            use_loc = true;
+        if( loc->has_flag( "ALLOWS_REMOTE_USE" ) ) {
```

Comment:
This will be true if the item has the flag, regardless of *where* the item currently is. That means: it does not actually check *where* the item is (map/inventory). Is this the intended check?

Reply:
I believe so.  Items with this flag are able to be activated from anywhere and are not meant to be picked up.  For example, a kiln.